smithy-codegen-core/src/main/java/software/amazon/smithy/codegen/core/directed/CodegenDirector.java

```diff
@@ -455,14 +455,11 @@ public Void stringShape(StringShape shape) {
             return null;
         }
 
-        /*
-        TODO: uncomment in idl-2.0 and remove the stringShape method
```

Comment:
Note, this comment said `remove the stringShape method`, but I think it needs to remain, because a code generator could chose to not `performDefaultCodegenTransforms` and thus need to handle string enums.